doc/modules/outlier_detection.rst

```diff
@@ -131,6 +131,10 @@ sections hereunder.
     :class:`neighbors.LocalOutlierFactor` and
     :class:`covariance.EllipticEnvelope`.
 
+  * See :ref:`sphx_glr_auto_examples_miscellaneous_plot_outlier_detection_bench.py`
+    for an example showing how to evaluate outlier detection estimators
+    using ROC curves.
+
```

Comment:
Could you please add a tag `.. _local_outlier_factor:` in (current) line 317 of this file i.e. above the corresponding subsection? (Similar to the tag `.. _isolation_forest:` in line 252). Then, call this reference at the beginning of the example.